lib/events/api.go

```diff
@@ -660,6 +660,30 @@ const (
 	// OktaAssignmentCleanupEvent is emitted when an assignment is cleaned up.
 	OktaAssignmentCleanupEvent = "okta.assignment.cleanup"
 
+	// AccessListCreateEvent is emitted when an access list is created.
+	AccessListCreateEvent = "access_list.create"
+
+	// AccessListUpdateEvent is emitted when an access list is updated.
+	AccessListUpdateEvent = "access_list.update"
+
+	// AccessListDeleteEvent is emitted when an access list is deleted.
+	AccessListDeleteEvent = "access_list.delete"
+
+	// AccessListReviewEvent is emitted when an access list is reviewed.
+	AccessListReviewEvent = "access_list.review"
+
+	// AccessListMemberCreateEvent is emitted when a member is added to an access list.
+	AccessListMemberCreateEvent = "access_list.member.create"
+
+	// AccessListMemberUpdateEvent is emitted when a member is updated in an access list.
+	AccessListMemberUpdateEvent = "access_list.member.update"
+
+	// AccessListMemberDeleteEvent is emitted when a member is deleted from an access list.
+	AccessListMemberDeleteEvent = "access_list.member.delete"
+
+	// AccessListMemberDeleteAllMembersEvent is emitted when all members are deleted from an access list.
+	AccessListMemberDeleteAllMembersEvent = "access_list.member.delete_all_members"
```

Comment:
```suggestion
	AccessListMemberDeleteAllMembersEvent = "access_list.member.delete_all"
```

For consistency. We don't call the delete event `access_list.member.delete_member`.

Reply:
My concern here (though reading it back over, I realize the name wasn't very good to begin with) is that there *is* a separate method `DeleteAllAccessListMembers` which deletes literally everything (used for cache reasons), as opposed to `DeleteAllAccessListMembersForAccessList` which deletes every member for an access list. I wanted to make this distinction so that anybody reading it didn't conflate the two.

What do you think about `delete_all_for_access_list`?

Reply:
I see. I still think `delete_all` is fine. The godoc above it is pretty clear as to what's going on.